alpha/apps/kaltura/lib/batch2/bcdl/kBusinessPostConvertDL.php

```diff
@@ -342,7 +342,8 @@ public static function handleConvertFinished(BatchJob $dbBatchJob = null, flavor
 		
 		if(count($childJobs) > 1 && count($waitingFlavorAssets) < 1)
```

Comment:
In your use case, you won't get into this code because of the condition `count($waitingFlavorAssets) < 1`.
Since you check the flavors readyness in the closer, you can remove this entire code entirely.